presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -266,6 +266,86 @@ public void testInnerJoinWithNullOnBothSides(boolean parallelBuild, boolean prob
     }
 
     @Test(dataProvider = "hashEnabledValues")
+    public void testProbeOuterJoinWithEmptyLookupSource(boolean parallelBuild, boolean probeHashEnabled, boolean buildHashEnabled)
```

Comment:
Can you add a test with an inner join and an empty lookup source, where you `new` up the operator and then it immediately says "finished"?